content/python/concepts/sets/sets.md

```diff
@@ -0,0 +1,115 @@
+---
+Title: "Sets" 
+Subjects:
+  - "Computer Science"
+  - "Data Science"
+Tags: 
+  - "Sets"
+  - "Data Types"
+CatalogContent: 
+  - "learn-python-3"
+  - "paths/computer-science"
+---
+
+A set is an unordered collection of elements with any duplicates.
```

Comment:
Wait, I thought sets don't have duplicates.

Reply:
Thanks for catching this typo! I changed the with to without.